plugins/filter/json_patch.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Stanislav Meduna (@numo68)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, annotations, division, print_function
+from json import loads
+from typing import TYPE_CHECKING
+from ansible.errors import AnsibleError, AnsibleFilterError, AnsibleOptionsError
+from ansible.module_utils.common.text.converters import to_native
+
+__metaclass__ = type  # pylint: disable=C0103
+
+if TYPE_CHECKING:
+    from typing import Callable
+
+try:
+    import jsonpatch
+
+    HAS_LIB = True
+except ImportError:
+    HAS_LIB = False
+
+
+class FilterModule:
+    """Filter plugin."""
+
+    def json_patch(
+        self, inp: object, op: str, path: str, value: object = None, **kwargs: dict
+    ) -> object:
+
+        if not HAS_LIB:
+            raise AnsibleError(
```

Comment:
Better use `AnsibleFilterError` for errors reported by filters:
```suggestion
            raise AnsibleFilterError(
```